components/shared/Text.js

```diff
@@ -15,12 +17,27 @@ export default props => {
     fontWeight = 'Bold';
   }
 
+  if (props.XL) {
```

Comment:
can define the size of text by passing in a prop. maybe we should handle this differently later, but this should be cool for now 😎